### PR TITLE
Info Bar component fetching data from the server

### PR DESCRIPTION
**Pull request by tawfiknasser**
**in this PR** 
we worked on both , front-end and back end .
**in front-end :** 
- we set setInfoBarVars function in the boxViewContainer which send request to the server to fetch the data.
- we passed the function to the card component so when we click on the card the info bar change.
**in the back-end :**
- created new file in the route folder which will contain all the userOnBox routes
- created route for each request
- implement the queries inside the routes 
_____
Issue #76

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,6 @@
     "express": "~4.16.0",
```
[the remark this answers is not in the document]
Comment (tawfiknasser) [reply package.json]: i'll :+1:

**Comment by MynahMarie** [comment]
Please see my comment about eslint on issue #28 , I think I found a configuration that works for fixing unwanted linting errors 😄 You can either make a separate PR for it or include the changes in this one by updating the PR description and adding a reference to the issue, whichever you prefer ☺️

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (tawfiknasser) [reply package.json]: #83

File: server/routes/index.js [hunk 2]
```diff
@@ -1,34 +1,11 @@
...
-    if (error) {
-      res.status(500).end();
-    } else {
-      res.json(result.rows);
-    }
-  });
-
-});
-router.post("/login", (req, res) => {
+router.post('/login', (req, res) => {
```
[the remark this answers is not in the document]
Comment (tawfiknasser) [reply server/routes/index.js]: login handler had been added in the next PR :+1:

**Pull request merged**